custom/opm/reports.py

```diff
@@ -500,7 +500,7 @@ def join_rows(self, rows):
         if len(rows) == 1:
             return rows[0]
         def zip_fn((i, values)):
-            if i == self.column_index('num_children'):
+            if i == self.column_index('num_children') or i == self.column_index('year_end_bonus_cash'):
```

Comment:
apologies, i think @esoergel is offline. can you clarify how this addresses the issue? I just don't remember exactly how these reports work